nixos/modules/services/databases/redis.nix

```diff
@@ -50,15 +50,9 @@ in
         description = "Which Redis derivation to use.";
       };
 
-      user = mkOption {
-        type = types.str;
-        default = "redis";
-        description = "User account under which Redis runs.";
-      };
-
       pidFile = mkOption {
         type = types.path;
-        default = "/var/lib/redis/redis.pid";
+        default = "/run/redis/redis.pid";
```

Comment:
I suggest removing the `pidFile` option entirely as there is no code to ensure the directory exists or the `redis` user has permissions to write there, which could also cause breakage.

Reply:
right - I doubt anybody is using pidfiles…